qiskit_ibm_runtime/estimator.py

```diff
@@ -427,3 +427,19 @@ def set_options(self, **fields: Any) -> None:
             **fields: The fields to update the options
         """
         self._options = Options._merge_options(self._options, fields)
+
+    @classmethod
+    def _validate_circuits(
+        cls,
+        circuits: Sequence[QuantumCircuit] | QuantumCircuit,
+    ) -> tuple[QuantumCircuit, ...]:
+        circuits = super()._validate_circuits(circuits)
+        for i, circuit in enumerate(circuits):
+            # Note: we need to remove the following check when Runtime Estimator
+            # supports dynamic circuits.
+            if circuit.num_clbits > 0:
+                raise ValueError(
+                    f"The {i}-th circuit has some classical bits. "
+                    "Estimator accepts quantum circuits without classical bits."
```

Comment:
```suggestion
                    f"The {i}-th circuit has {circuit.num_clbits} classical bits. "
                    "Estimator only accepts quantum circuits without classical bits."
```
Rather than saying "some" we can be more precise which may be helpful

Reply:
Thanks. Your suggestion is more comprehensive.